src/app/loadout-builder/process/useProcess.ts

```diff
@@ -222,20 +228,27 @@ const groupComparator = chainComparator(
 );
 
 /**
- * This groups items for process depending on whether any general, other or raid mods are locked as follows
- * - If there are general, other or raid mods locked it groups items by (stats, masterworked, modSlot, energyType).
- * - If there are only general mods locked it groupes items by (stats, masterwork, energyType)
- * - If no general, other or raid mods are locked it groups by (stats, masterworked).
+ * To reduce the number of items sent to the web worker we group items by a number of varying
+ * parameters, depending on what mods and armour upgrades are selected.
+ *
+ * After items have been grouped we only send a single item (the first one) as a representitive of
+ * said group. All other grouped items will be available by the swap icon in the UI.
  *
- * Note that assumedMasterwork effects this.
+ * It can group by any number of the following concepts depending on locked mods and armor upgrades,
+ * - Stat distribution
```

Comment:
This is only "exactly equal stats" right? I suppose we could also filter out "strictly worse stats", I'll file a separate issue.

Reply:
Yes it is, that would actually be a good optimisation. I hadn't thought of that. It would be seperate to this I think in that we could just ignore those items, it wouldn't make sense to add them to these groups as they might lower stat tiers.